backend/Fields/BatchPostFields.ts

```diff
@@ -0,0 +1,55 @@
+import { FieldAttributes, FieldModel } from "../../models/field.js";
+import { NounAttributes, NounModel } from "../../models/noun.js";
+import { sequelize } from "../DB.js";
+import { router } from "../Router.js";
+import { invalidRequest, notFound } from "../Utils/EndpointResponses.js";
+import { body, validationResult, param } from "express-validator";
+
+router.post<Params, ResponseBody, RequestBody>(
+  "/api/nouns/:nounId/fields",
+  body("fields").isArray(),
+  body("fields.*.type").isString().trim().toLowerCase(),
+  body("fields.*.columnName").isString().trim(),
+  body("fields.*.friendlyName").isString().trim(),
+  body("fields.*.activeStatus").isBoolean(),
+  param("nounId").isInt().toInt(),
```

Comment:
Do we need the `toInt()` if the nounId param is already an int?

Reply:
We do - all express params start out as strings, as they're just part of the URL and express doesn't know which parts are strings/numbers/dates/etc